journal/api/src/db/drizzle/0001_stiff_grandmaster.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE `location` (
```

Comment:
stiff grandmaster....you know I'm 12 right? *giggle*

Reply:
Love it!